src/main/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringcreateanorderapi/service/courthearing/HearingEventHandler.kt

```diff
@@ -652,10 +652,13 @@ class HearingEventHandler(
   ): InterestedParties {
     return InterestedParties(
       versionId = versionId,
+      notifyingOrganisation = notifyingOrganisation,
```

Comment:
This should be mapped to notifyingOrganisationName instead, I'll make the change in my mapping branch